### ISSUE TITLE
How to install the add-in?

### ISSUE THREAD

**Issue by it19862** [stevencohn/OneMore#28]
How to install the add-in?
How do I add an add-in to OneNote?

**Comment by stevencohn** [comment]
Look for the latest installer here: https://github.com/stevencohn/OneMore/releases

**Comment by it19862** [comment]
@stevencohn 

I have little experience, could you please describe the steps.

**Description**
I pressed F6.
I ran "..\OneMoreSetup\Debug\setup.exe"
Result: Error.
 
**Error**
_Al error occurred attempting to install One More Add In.
Unable to locate application file 'Setupx86.msi'.
 
See the setup log file located at 'C: \ Users \ admin \ AppData \ Local \ Temp \ VSD8852.tmp \ install.log' for more information._
 
**I use:**
Win-7x64
VS - 2019
NoteHighlight2016 - I installed the "NoteHighlight2016" add-on, then uninstalled it.

**Picture-1**
![2020-08-14_21-07-56](https://user-images.githubusercontent.com/18245828/90281118-c02d7a80-de74-11ea-9f3f-7d78e1addce5.png)

**Picture-2**
![2020-08-14_21-09-21](https://user-images.githubusercontent.com/18245828/90281132-c4f22e80-de74-11ea-87ec-6bee067cf9fd.png)

**Picture-3**
![2020-08-14_21-09-31](https://user-images.githubusercontent.com/18245828/90281146-ca4f7900-de74-11ea-877b-51f08b975c1b.png)

**Comment by it19862** [comment]
@stevencohn 
I solved the installation problem.

I realized what my mistake was.
My mistake:
I downloaded the project from the main page https://github.com/stevencohn/OneMore and tried to install from this project.
Solution: download from the link https://github.com/stevencohn/OneMore/releases and upgrade the OneMore_2.7.4_Setupx86.msi file


Question-2.
I ran the OneMore_2.7.4_Setupx86.msi file.
The add-in has been installed.
Result: The add-in does not appear in the ribbon.

I went to Parameters -> Add-ons -> Go.
I selected the add-in.
I went to Parameters -> Add-ons -> Go again.
Result: The add-in was not selected.

**Question:** why is the ribbon tab not showing?
